### ISSUE TITLE
New file in new subfolder does not show up on compare

### ISSUE THREAD

**Issue by KurtOplev** [WinMerge/winmerge#209]
I have a “left folder” and a “right folder”
I then create a “subfolder” in the “right folder”
I then create a new file in the “subfolder” in the “right folder”

Now when I want to compare what’s new I run WinMerge but do not get the expected result
Please see attached example
[WinMerge example.docx](https://github.com/WinMerge/winmerge/files/3744700/WinMerge.example.docx)

**Comment by sdottaka** [comment]
I think that it will work as expected when enabling the following options.

![image](https://user-images.githubusercontent.com/98126/67137159-aa0e5800-f26b-11e9-9c72-327c784f0ce5.png)

**Comment by KurtOplev** [comment]
Hello Takashi

 

Thank you very much – that did the trick.

 

Why this is not the default I do not understand though.

I did not look for such option, as I did not imagine the need for excluding contents.

 

Shall I do something to close the issue?

 

Best Regards, Kurt Pedersen

 

 

Fra: Takashi Sawanaka <notifications@github.com> 
Sendt: 19. oktober 2019 05:29
Til: WinMerge/winmerge <winmerge@noreply.github.com>
Cc: KurtOplev <mail@kurtkristianpedersen.dk>; Author <author@noreply.github.com>
Emne: Re: [WinMerge/winmerge] New file in new subfolder does not show up on compare (#209)

 

I think that it will work as expected when enabling the following options.

 <https://user-images.githubusercontent.com/98126/67137159-aa0e5800-f26b-11e9-9c72-327c784f0ce5.png>
